### DISCUSSION TITLE
Doge Name Service (DNS)

### DISCUSSION THREAD

**Discussion by srijanjain** [dogecoin/dogecoin#3399] [General]
Implementing Doge Name Service (DNS) .doge similar to[ Ethereum Name Service (ENS) ]([url](https://ens.domains/)) .eth for increasing the adaptability and usability of the doge ecosystem by the general public. 

Examples: father.doge,  srijan.doge ,etc.

**Comment by ThisIsMyDogeAccount** [comment]
To be fair I do not even think most browsers adequately acknowledge .eth as a domain and just end up going to the built in search engine.

I could be wrong.

**Comment by Purukitto** [comment]
This could helpful to aid development of DApps based on Doge

**Comment by xiangbobozhenghao** [comment]
This is a great idea, and developers should

**Comment by sabotagebeats** [comment]
> To be fair I do not even think most browsers adequately acknowledge .eth as a domain and just end up going to the built in search engine.
> 
> I could be wrong.

I believe brave supports it https://brave.com/brave-crypto-dns-strategy/ 

I personally have not done the research and am just throwing this out there but  since  dogecoin is already forking namecoin auxpow I believe this would just be another section of namecoin to fork in order to support this.
